packages/migrations/src/migration.ts

```diff
@@ -1,15 +1,43 @@
 import * as wrappers from '@0x/abi-gen-wrappers';
 import { ContractAddresses } from '@0x/contract-addresses';
 import * as artifacts from '@0x/contract-artifacts';
-import { assetDataUtils } from '@0x/order-utils';
 import { Web3ProviderEngine } from '@0x/subproviders';
-import { BigNumber, providerUtils } from '@0x/utils';
+import { AbiEncoder, BigNumber, providerUtils } from '@0x/utils';
 import { Web3Wrapper } from '@0x/web3-wrapper';
-import { SupportedProvider, TxData } from 'ethereum-types';
+import { MethodAbi, SupportedProvider, TxData } from 'ethereum-types';
 import * as _ from 'lodash';
 
 import { erc20TokenInfo, erc721TokenInfo } from './utils/token_info';
 
+// HACK (xianny): Copied from @0x/order-utils to get rid of circular dependency
+/**
+ * Encodes an ERC20 token address into a hex encoded assetData string, usable in the makerAssetData or
+ * takerAssetData fields in a 0x order.
+ * @param tokenAddress  The ERC20 token address to encode
+ * @return The hex encoded assetData string
+ */
+function encodeERC20AssetData(tokenAddress: string): string {
```

Comment:
@xianny we have `DevUtils` contract now in `abi-gen-wrappers` that has this exact method. Why not just use that?

Reply:
https://github.com/0xProject/0x-monorepo/pull/2108 isn't merged yet. Without that, we still need to provide the address to a deployed DevUtils Contract, which doesn't exist before migrations run. We need to encode zrx asset data to call `DevUtilsContract.deployFromZeroExArtifact`. 

I can change it after merging #2108

Reply:
ok!